Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -1580,6 +1580,8 @@ DECLARE_SOA_DYNAMIC_COLUMN(GetHepMCStatusCode, getHepMCStatusCode, //! The HepMC
                            [](uint8_t flags, int statusCode) -> int { if ((flags & o2::aod::mcparticle::enums::ProducedByTransport) == 0x0) { return o2::mcgenstatus::getHepMCStatusCode(statusCode); } else { return -1; } });
 DECLARE_SOA_DYNAMIC_COLUMN(IsPhysicalPrimary, isPhysicalPrimary, //! True if particle is considered a physical primary according to the ALICE definition
                            [](uint8_t flags) -> bool { return (flags & o2::aod::mcparticle::enums::PhysicalPrimary) == o2::aod::mcparticle::enums::PhysicalPrimary; });
+DECLARE_SOA_DYNAMIC_COLUMN(PVector, pVector, //! Momentum vector in x,y,z-directions in GeV/c
+                           [](float px, float py, float pz) -> std::array<float, 3> { return std::move(std::array<float, 3>{px, py, pz}); });
```

Comment:
`move` in return statement is redundant, since the return is by-value.

Reply:
So there is an additional copy? Any way to avoid it?

Reply:
for 3 floats, the copy is probably better than dealing with a pointer

Reply:
@jgrosseo there is so called return value optimization, the copy will happen once - the array will be constructed in the place which calls the function. And indeed, copying small numbers of floats is preferrable, it can be optimized in the machine code.

Reply:
Thanks! So then we remove the `std::move`. @vkucera This is also in a second place where we have the pVector already, right?